src/ProbeMeasurements.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+#define WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+
+#include <base/Float.hpp>
+#include <base/Temperature.hpp>
+#include <base/Time.hpp>
+
+namespace water_probe_acquanativa_ap3 {
+    struct ProbeMeasurements {
+        base::Time time;
+        
+        /** Water dissolved oxygen concentration in Kg/L */
+        float concentration = base::unknown<float>();
+
+        /** Water dissolved oxygen % saturation */
+        float saturation = base::unknown<float>();
+
+        /** Water temperature in K */
+        base::Temperature temperature;
+
+        /** Water pH */
+        float ph = base::unknown<float>();
+
+        /** Water conductivity in S/cm*/
+        float conductivity = base::unknown<float>();
+
+        /** Water salinity in PSU */
+        float salinity = base::unknown<float>();
+
+        /** Water dissolved solids concentration in ppm */
+        float dissolved_solids = base::unknown<float>();
```

Comment:
Normalize. ppm is a human-centric unit because we don't like having to read 10e-6 or even worse values like 0.0000001. Computers don't care. Programmers do care to not have to read every single bloody API in detail to know what unit has been chosen.

ppm should be converted to value * 1e-6